src/components/PanelHeader/PanelHeader.css

```
@@ -262,13 +262,21 @@
           line-height: 56px;
           height: 56px;
           font-weight: 500;
-          }
+          display: flex;
+          align-items: center;
+        }
 
         .PanelHeader--android .PanelHeader-content > *:not(.Search):not(.PanelHeaderContent) {
           padding: 0 6px;
-          height: 56px;
-          line-height: 56px;
-          }
+          height: auto;
+          line-height: auto;
+          max-width: 100%;
+          flex: 1;
+          min-width: 0;
```

Скорее всего не нужно

---

```suggestion
          flex: 1;
          min-width: 0;
```
Это нужно, чтобы корректно работала обрезка текста. Без этого не будет появляться многоточие.